cmake/ctypesgen.cmake

```diff
@@ -58,10 +58,14 @@ foreach(HDR ${HDRS})
   list(APPEND HEADERS "${OUTDIR}/${GRASS_INSTALL_INCLUDEDIR}/grass/${HDR}")
 endforeach()
 
+# set(RUNTIME_GISBASE_1
+#     "\"${RUNTIME_GISBASE}\""
+# )
+
 message(STATUS "Generating ${OUT_FILE}")
 execute_process(
   COMMAND
-    ${PYTHON_EXECUTABLE} ${CTYPESGEN_PY} --cpp "${CTYPESFLAGS}"
+    ${PYTHON_EXECUTABLE} ${CTYPESGEN_PY} --cpp ${CTYPESFLAGS}
```

Comment:
I don't think so because `cl.exe -E` needs to be passed to `--cpp` "as is" including the space. Without double quotes, they would be treated as separate arguments. Including your changes above, I guess you created this PR without merging the latest main branch?